src/controllers/mysql/queries/postAndCommentsQueries.ts

```diff
@@ -0,0 +1,65 @@
+import { PostObject } from '../../../types/post'
+import connection from '../connection';
+import { resolve } from 'url';
+
+
+export function selectAllPostAndComments_ByUser_name(username:string){
```

Comment:
Consistant JS functions names, use camel case only (also better name?)
selectAllContentByUsername

Reply:
Totally understood. Gone through all myu functions and renamed to follow the convention and tried to make them easier to understand with shorter names and added comments above them